app/exporters/odt/cv.rb

```diff
@@ -4,12 +4,19 @@
 module Odt
   class Cv
 
-    def initialize(person)
+    def initialize(person, params)
       @person = person
+      @params = params
     end
 
+    def anon?
+      @params[:anon].present? && @params[:anon] == 'true'
+    end
```

Comment:
Move this below the private keyword so it's a private method
You can also refactor the statement to `@params[:anon].presence == 'true'`